### DISCUSSION TITLE
Windows ARM64 build

### DISCUSSION THREAD

**Discussion by makanys** [obsproject/obs-studio#11094] [General]
Hi,

How can I build Windows ARM64 version from source? 

Thanks
Sandor

**Comment by sebastian-s-beckmann** [comment]
You cannot currently build a Windows ARM64 version from source.
Work for that is currently ongoing but no ETA, feel free to use the x86_64 version with translation in the meantime.

**Reply by makanys** [reply]
Thank you. If you need to tester. I am at your disposal.

Thanks
Sandor
